samples/test/utils/clean.js

```diff
@@ -0,0 +1,80 @@
+// Copyright 2020 Google LLC
```

Comment:
no need to create a utils directory, just put this in the `/test` folder

Reply:
Done